src/app/api/axios.jsx

```diff
@@ -1,10 +1,18 @@
 import React, { createContext, useContext, useMemo } from 'react';
 import Axios from 'axios';
+import { 
+  applyAuthTokenInterceptor, 
+  getAccessToken }
+  from 'axios-jwt';
 import { element } from 'prop-types';
 
+import { getAngularService } from 'services/angular-react-helper';
+
 const AxiosContext = createContext();
 
 function AxiosProvider({ children }) {
+  const ngLocalStorage = getAngularService('$localStorage');
```

Comment:
In other places where we pull in an angular service, we continue to use the same name for that service, so this line would be:

```suggestion
  const $localStorage = getAngularService('$localStorage');
```